# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1437068701 14400
#      Thu Jul 16 13:45:01 2015 -0400
# Node ID fb97c11f8e676d594aba630f1e57bdf9b813fc11
# Parent  9df3feddcf22150dd1452f5d855a34c68c46b1ff
clean generated AUTHORS file

* doc/module.mk (MAINTAINERCLEANFILES): Include AUTHORS in the list.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -307,17 +307,17 @@ octetc_DATA += \
   doc/interpreter/macros.texi
 
 DOCSTRING_FILES = $(shell $(srcdir)/doc/interpreter/find-docstring-files.sh "$(top_srcdir)")
 
 DOCSTRING_DEPS = scripts/.DOCSTRINGS libinterp/.DOCSTRINGS
 
 doc/interpreter/doc-cache: $(DOCSTRING_DEPS) $(OCTAVE_INTERPRETER_TARGETS) doc/interpreter/mk_doc_cache.m doc/interpreter/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f $@-t $@ && \
-	$(top_builddir)/run-octave -f -q -H $(srcdir)/doc/interpreter/mk_doc_cache.m - $(srcdir)/doc/interpreter/macros.texi $(DOCSTRING_FILES) >$@-t && \
+	$(top_builddir)/run-octave -f -q -H $(srcdir)/doc/interpreter/mk_doc_cache.m - $(srcdir)/doc/interpreter/macros.texi $(DOCSTRING_FILES) > $@-t && \
 	mv $@-t $@
 
 $(MUNGED_TEXI_SRC): $(DOCSTRING_DEPS) $(munge_texi_SOURCES)
 
 %.texi : %.txi doc/interpreter/munge-texi.pl doc/interpreter/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f $@-t $@ && \
 	$(PERL) $(srcdir)/doc/interpreter/munge-texi.pl $(top_srcdir) $(DOCSTRING_FILES) < $< > $@-t && \
 	mv $@-t $@
@@ -374,16 +374,17 @@ interpreter-clean:
 interpreter-maintainer-clean:
 	rm -f $(BUILT_OCTAVE_TEXI_SRC)
 
 DISTCLEANFILES += \
   $(BUILT_TEXINFOS) \
   $(OCTAVE_HTML_STAMP)
 
 MAINTAINERCLEANFILES += \
+  AUTHORS \
   $(BUILT_DOC_IMAGES) \
   doc/interpreter/doc-cache
 
 DIRSTAMP_FILES += doc/interpreter/$(octave_dirstamp)
 
 liboctave_TEXINFOS = \
   doc/liboctave/array.texi \
   doc/liboctave/bugs.texi \
